CONTRIBUTING.rst

```diff
@@ -432,6 +432,19 @@ To run lint checks against JavaScript/TypeScript sources, execute:
 
     $ make dev/jslint
 
+Formatting your code
+^^^^^^^^^^^^^^^^^^^^
+
+We use prettier to enforce code formatting for all of our TypeScript and less files.
+To automatically format your Angular code run:
+
+.. code-block:: console
+
+    $ make dev/prettier
+
+Unformatted code will cause the travis build to fail when you push your changes to
+GitHub.
```

Comment:
We need to mention that there is a `.prettierrc.yaml`, where settings live.

Reply:
Also, might be a good idea to add a link to https://prettier.io/docs/en/editors.html, so people have some idea of how to configure their editor.